features/polyExplorer/src/data/products.json

```diff
@@ -11947,7 +11947,7 @@
             "messenger"
         ],
         "productOwner": [
-            "TikTok Technology limited (IE)",
+            "TikTok Technology Limited (IE)",
```

Comment:
Did you change this *by hand*? If we are going to need and to this every time, wouldn't it be better to include a patching script that does this (and maybe other stuff) for us?

Reply:
No I didn't. @tstec-polypoly changed it on the polypedia backend

Reply:
OK, not our turf, but still. Manual editing is expensive, be it done here or there.

Reply:
The data that we present is manually researched data. Since the input is done manually the mistakes need to be fixed manually

Reply:
So this is a once-only fix, which has already been fixed in origin, right? What I meant is that if once produced the JSON it needs to be edited manually, it would be better to have a patching script. But I see that's not the case.

Reply:
Yeah and sadly we do have a patching script in place :D